src/main/java/visualization/Visualization.java

```diff
@@ -77,4 +77,20 @@ public static void getImage(Model model) {
             vs.frame.dispose();
         }
     }
-}
\ No newline at end of file
+
+
```

Comment:
Copy paste funkcji Kamila i Łukasza z tym, że zwraca stringa

Reply:
To samo, można im zmienić myślę po prostu :)